dvclive 1.0 (#4085)
iterative/dvc.org: content/docs/dvclive/api-reference/index.md

live = Live()
```

Or use as a context manager:

```python
with Live() as live:
```

> 

Reviewer: Not blocking (but probably would block if I wasn't trying to get this out today): can we remove the suggested usage without the context manager? It could result in unexpected behavior.

IMO there are 2 categories of people who will use it without the context manager:
1. People who can figure out themselves that they can omit it and understand the consequences of doing so (they don't need this documented).
2. People who won't be able to figure out this pattern for themselves but may also be surprised at the consequences (they probably should use the context manager).

Reviewer: Also, can we avoid mentioning `context manager`? I think it's too technical.

Author: Makes sense.

live.next_step()
```

See `Live.next_step()`.

Under the hood, `Live.next_step()` calls `Live.make_summary()` and
`Live.make_report()`.

> If you want to decouple the `step` update from the rest of the calls, you can
> manually modify the `Live.step` property and call `Live.make_summary()` /
> `Live.make_report()`.

Reviewer: Non-blocking suggested rewording:

```suggestion
If you want to update the `step` to a different number or don't want to call
`Live.make_summary()` / `Live.make_report()`, you can manually modify the
`Live.step` property (for example, `live.step = 1`).
```

Also, why no mention of `make_checkpoint()`?

Reviewer 2: > Why no mention of make_checkpoint()?

I'm also curious as to why this is not considered part of the https://dvc.org/doc/dvclive/api-reference#basic-workflow .
